docs: update challenge (#8)
juntossomosmais/frontend-challenge: README.md

## The Challenge

> ### About layout

Reviewer: **suggestion (non-blocking):** 
```suggestion
### About the layout
```

The [following Figma layout needs to be developed](https://www.figma.com/file/ED8oK1CyH2BD5BjSAcy9vF/Teste-Front-end?node-id=505%3A2613).

> <img src="layout-jsm.png" width="800">
> 
> It's important to say that it is just a prototype, the idea is to check your ability to propose improvements, features and contribute to the product UI 😄.
> 
> The logo is the same as the one on the repository.

Reviewer: **suggestion (non-blocking):** my friend, what do you think of the following structure:

```suggestion
The logo is the same as the one in the repository.

It's important to say that it is just a prototype! We would also like to see your ability to propose improvements, features and new contributions to the product's UI 😄!
```

> **Feel completely free** to make improvements like:

Reviewer: **suggestion (non-blocking):** 
```suggestion
**Feel completely free** to make improvements such as:

- Adding new functionalities;
- Creating animations;
- Developing visual optimizations;
- etc.
```

- New functionalities
- Animations
- Visual optimizations
- etc.

You also can add new filters if you want 😋

### About API

The JSON that you need to develop for this application is available at:

- `https://jsm-challenges.s3.amazonaws.com/frontend-challenge.json`

> You can feel free to use BFF (Back-end for Front-end) before using it on client-side, if you think that if it makes sense 👀

Reviewer: **suggestion (non-blocking):**
```suggestion
Feel free to use BFF (Back-end for Front-end) before using it on the client-side, if you think that if it makes sense 👀!
```

## The Challenge

### About the layout

The [following Figma layout needs to be developed](https://www.figma.com/file/ED8oK1CyH2BD5BjSAcy9vF/Teste-Front-end?node-id=505%3A2613).

The logo is the same as the one in the repository.

It's important to say that it is just a prototype! We would also like to see your ability to propose improvements, features and new contributions to the product's UI 😄!

**Feel completely free** to make improvements such as:

- Adding new functionalities;
- Creating animations;
- Developing visual optimizations;
- etc.

- New functionalities
- Animations
- Visual optimizations

> - etc.

Reviewer: **issue:** isn't this part being repeated?

If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue.

> Please, check if your question has already been answered in [an issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) 😌

Reviewer 2: **nitpick:** 
```suggestion
Please, check if your question has already been answered on [an issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) 😌
```

> If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue.

Reviewer: **suggestion:** 
```suggestion
If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue. But, before that, please, make sure that your question hasn't been answered on a previous [issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=)
```
